src/SeleniumLibrary/__init__.py

```diff
@@ -402,26 +411,39 @@ def browser(self):
         return self.driver
 
     def find_element(self, locator, parent=None):
-        """Find element matching ``locator``.
-
-        This method and :meth:`find_elements` form the recommended
-        public API for external tools to get elements via SeleniumLibrary.
+        """Find element matching `locator`.
 
         :param locator: Locator to use when searching the element.
             See library documentation for the supported locator syntax.
-        :param parent: Optional parent ``WebElememt`` to search child elements
-            from. By default search starts from the root using ``WebDriver``.
+        :param tag: Limit searching only to these elements.
+        :param required: Raise `ElementNotFound` if element not found when
+            true, return `None` otherwise.
+        :param parent: Optional parent `WebElememt` to search child elements
+            from. By default search starts from the root using `WebDriver`.
+        :type parent: selenium.webdriver.remote.webelement.WebElement
+        :return: Found `WebElement` or `None` if element not found and
+            `required` is false.
         :rtype: selenium.webdriver.remote.webelement.WebElement
-        :raises SeleniumLibrary.errors.ElementNotFound: If element not found.
+        :raises SeleniumLibrary.errors.ElementNotFound: If element not found
+            and `required` is true.
         """
         return self._element_finder.find(locator, parent=parent)
 
     def find_elements(self, locator, parent=None):
-        """Find all elements matching ``locator``.
+        """Find all elements matching `locator`.
 
-        Returns a list of ``WebElement`` objects. If no matching elements
-        are found, the list is empty. Otherwise semantics are exactly same
-        as with the :meth:`find_element` method.
+        :param locator: Locator to use when searching the element.
+            See library documentation for the supported locator syntax.
+        :param tag: Limit searching only to these elements.
+        :param parent: Optional parent `WebElememt` to search child elements
+            from. By default search starts from the root using `WebDriver`.
+        :type parent: selenium.webdriver.remote.webelement.WebElement
+        :return: list of found `WebElement` or `[]` if element not found.
+        :rtype: list[selenium.webdriver.remote.webelement.WebElement]
+
+        Always returns a list of `WebElement` objects. If no matching element
+        is found, the list is empty. Otherwise semantics are exactly same
+        as with :meth:`find_element`.
```

Comment:
This is duplicate information now that arguments are explained above and can be removed.